docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
     working_dir: /home/node/app
     volumes:
       - ./server/:/home/node/app
-    command: ash -c "npm i && node index.js"
+    command: ash -c "sed -i 's/stun:stun.l.google.com:19302/stun:${STUN_SERVER_DOMAIN}:${STUN_SERVER_UDP_PORT}/g' client/scripts/network.js && npm i && node index.js"
```

Comment:
if `docker-compose up` is executed without defining those variables explicitly it fails. I guess a good way would be to leave the current google stun server as a default and only install the coturn image if these variables are set